### PR TITLE
Add none check for Cleveland `image_data`

### PR DESCRIPTION
**Pull request by stacimc**
## Description

This error was hit in a recent Cleveland Dag run:
```
  File "/usr/local/airflow/openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum.py", line 76, in _get_image_type
    if keyed_image := image_data.get(key):
AttributeError: 'NoneType' object has no attribute 'get'
```

We already handle the situation where `images` is not available in the response, but if it is explicitly set to `None` then `_get_image_type` is called with None. This PR just adds a check and a new test.

## Testing Instructions

`just test`.

You can also try running Cleveland locally with the query params that were being run when the error was hit in production. Manually trigger with this dagrun conf:
```
{"initial_query_params": {"cc": "1", "has_image": "1", "limit": 1000, "skip": 16000}}
```
It's sufficient to make sure it gets past that batch.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by AetherUnbound — APPROVED** [review]
DAG ran great locally with the initial params you supplied!

File: openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum.py [hunk 1]
```diff
@@ -72,6 +72,9 @@ def get_record_data(self, data):
     @staticmethod
     def _get_image_type(image_data):
         # Returns the image url and key for the image in `image_data` dict.
```
Comment (krysal) [inline openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum.py:74]: The function name is confusing if it's doing what the comment says 🤔 By the tests looks like it's actually _not_ returning the image type/file extension but a bunch of other stuff.

Can we update the name? Maybe not in this PR but letting the note.

**Review by krysal — APPROVED** [review]
The change to fix the error looks good to me 👍 I found something a bit odd about the function in question.

File: openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum.py (hunk 1 above)
Comment (stacimc) [reply openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum.py]: Good point! I think the original function name must have referred to the fact that we're selecting an image from the `image_data` dict, _by type_ (web/print/full)? 😮 That's _definitely_ confusing. Updated!

**Pull request merged**